src/middlewared/middlewared/main.py

```diff
@@ -275,19 +290,6 @@ async def on_close(self, *args, **kwargs):
         self.middleware.unregister_wsclient(self)
 
     async def on_message(self, message):
-        if message.get('msg') == 'method' and message.get('method') and isinstance(message.get('params'), list):
```

Comment:
I am torn over this, if UI sends a malformed response for some reason and we send an error ( which we log in the dequeue ) but we have not logged the incoming call to which it was replied, we potentially lose a debugging case ? Also in practice how many failed calls do really happen or in other words what improved metrics do we see wrt less resource consumption if we do not log the failure scenarios ?

Reply:
I don't really see a reason to log the error information in the `deque` if that's what you're implying 😄